packages/addons/addon-depends/chrome-depends/itstool/package.mk

```diff
@@ -2,10 +2,14 @@
 # Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="itstool"
-PKG_VERSION="2.0.6"
-PKG_SHA256="6233cc22726a9a5a83664bf67d1af79549a298c23185d926c3677afa917b92a9"
+#PKG_VERSION="2.0.6"
```

Comment:
@heitbaum can you remove/bump the versions that we don't include out commented stuff ?

Reply:
@CvH - fixed.